wgsl/index.bs

```diff
@@ -942,16 +945,7 @@ when:
     standard buffer layout rules for |C| when all the following are satisifed:
     * Each member type |T|<sub>|i|</sub> satisfies standard buffer layout rules for |C|
     * Members do not overlap, and are laid out in declaration order:
```

Comment:
should remove the trailing ":" here and below now as the examples are gone

Reply:
done